api/proto/config/appconfig.proto

```diff
@@ -54,4 +55,6 @@ message AppInstanceConfig {
     // Config flag if the app-instance should be made accessible
     // through a remote console session established by the device.
     bool remoteConsole = 12;
+    bytes cipherTextUserData = 13;
```

Comment:
Add comment for userData that this is not used on the wire (and will be deprecated??) and a cmment here that this is encrypted userData.